analyzers/src/SonarAnalyzer.CSharp/Rules/EnumNameHasEnumSuffix.cs

```diff
@@ -31,7 +31,5 @@ namespace SonarAnalyzer.Rules.CSharp
     public sealed class EnumNameHasEnumSuffix : EnumNameHasEnumSuffixBase<SyntaxKind>
     {
         protected override ILanguageFacade<SyntaxKind> Language { get; } = CSharpFacade.Instance;
-
-        public EnumNameHasEnumSuffix() : base(RspecStrings.ResourceManager) { }
     }
 }
```

Comment:
_Definition:_ This is the optimum amount of code needed for a rule with common base 😄  

I promise to stop deleting code from rules that reach the optimum 👼 🤣

Reply:
The optimum would be to use ISourceGenerator and generate all these "boilerplate" classes with just the LanguageFacade. :)